test/cases/api.sh

```diff
@@ -90,6 +90,11 @@ cat > "$REQUEST_FILE" << EOF
       "architecture": "$ARCH",
       "image_type": "ami",
       "repositories": $(jq ".\"$ARCH\"" /usr/share/tests/osbuild-composer/repositories/"$DISTRO".json),
+      "customizations": {
+        "packages": [
+          "postgres",
```

Comment:
Ah I think this final `,` might make the json of the request invalid :)

Reply:
Oops, yeah this is invalid.